jobs.groovy.j2

```diff
@@ -327,7 +327,8 @@ branches.each {
     }
 {# limit execution to jenkins slaves with a particular label                 #}
     label("{{ job_values.on_nodes_with_labels }}")
-    {{ wrappers(job_values, 'none') }}
+{% set repo_dir = "/tmp/workspace/${dashProject}/${dashBranchName}/" + _job_name + "/_trial_temp" %}
```

Comment:
:question: `repo_dir` seems like a bad name for this, since it isn't a repo. `directory_to_delete` perhaps?